test/e2e/app/run.go

```diff
@@ -42,55 +42,67 @@ func Deploy(ctx context.Context, def Definition, cfg DeployConfig) (types.Deploy
 	}
 
 	genesisValSetID := uint64(1) // validator set IDs start at 1
-	genesisVals, err := toPortalValidators(def.Testnet.Validators)
-	if err != nil {
-		return nil, nil, err
-	}
+	var genesisVals []bindings.Validator
```

Comment:
it almost feels like a `DeployMonitorOnly` is wroth the effort, that would result in less switching.

Reply:
Do you mean a separate deploy command?